web/admin/src/DailyChart.js

```diff
@@ -148,7 +148,7 @@ class DailyChart extends PureComponent {
         .catch(() => null),
     )
     return Promise.all(attendeeQuestionPromises).then(results => {
-      const users = results.filter(x => x)
+      const users = results.filter(x => x.id)
```

Comment:
This will fail if `x` is null.  When would `x` be defined, but not have an `id`?